Decred Wallet/Custom Views/ErrorBanner.swift

```diff
@@ -0,0 +1,71 @@
+//
+//  ErrorBanner.swift
+//  Decred Wallet
+//
+// Copyright (c) 2018-2019 The Decred developers
+// Use of this source code is governed by an ISC
+// license that can be found in the LICENSE file.
+
+import UIKit
+
+class ErrorBanner: UIView {
+    private var parent:UIViewController?
+    
+    init(parent: UIViewController) {
+        super.init(frame: .zero)
+        self.parent = parent
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    public func show(text:String) {
```

Comment:
Can this be more generic? like change the class name and add parameters for background color as well as size